docs_user/modules/openstack-ironic_adoption.adoc

```diff
@@ -8,20 +8,181 @@
+set.
+
+* [neutron]cleaning_network
+* [neutron]provisioning_network
+* [neutron]rescuing_network
+* [neutron]inspection_network
+* [conductor]automated_clean
+* [deploy]erase_devices_priority
+* [deploy]erase_devices_metadata_priority
+* [conductor]force_power_state_during_sync
+// FIXME: The setting above likely should be True by default in deployments, but would have been *false* by defaults on prior underclouds.
+
+The following parameters *can* be set individually on a node, however, some operators choose to use embedded configuration options to avoid the need to set it individually when creating/managing baremetal nodes. We recommend you check your prior ironic.conf file for these parameters, and if set apply as specific override configuration.
+
+* [conductor]bootloader
+* [conductor]rescue_ramdisk
+* [conductor]rescue_kernel
+* [conductor]deploy_kernel
+* [conductor]deploy_ramdisk
+
+Finally, a parameter which may be important based upon your configuration and experience, are the instances of `kernel_append_params`, formerly `pxe_append_params` in the `[pxe]` and `[redfish]` configuration sections. Largely this parameter is used to appy boot time options like "console" for the deployment ramdisk and as such often seeks to be changed.
+
+// TODO:
+// Conductor Groups?!
+
+As a warning, hardware types set via the `ironic.conf` `enabled_hardware_types` parameter and hardware type driver interfaces starting with `staging-` are not available to be migrated into an adopted configuration.
+
+Furthermore, Director based deployments made architectural decisions based upon self-management of services. When adopting deployments, you don't necessarilly need multiple replicas of secondary services such as the Introspection service. Should the host the container is running upon fail, OpenShift will restart the container on another host. The short-term transitory loss 
 
 == Procedure - Ironic adoption
 
-*TODO*
+=== Applying Initial Configuration
+
+As an initial step patch the OpenStackControlPlane to deploy Ironic:
+
+[source,yaml]
+----
+oc patch openstackcontrolplane openstack -n openstack --type=merge --patch '
+spec:
+  ironic:
+    enabled: true
+    template:
+      databaseInstance: openstack
+      ironicAPI:
+        replicas: 1
+      ironicConductors:
+      - replicas: 1
+        storageRequest: 10G
+        customServiceConfig: |
+          [neutron]
+          cleaning_network=<cleaning network uuid>
+          provisioning_network=<provisioning network uuid>
+          rescuing_network=<rescuing network uuid>
+          inspection_network=<introspection network uuid>
+          [conductor]
+          automated_clean=true
+      ironicInspector:
+        replicas: 1
+      ironicNeutronAgent:
+        replicas: 1
+      secret: osp-secret
+'
+----
+
+// TODO ? Do we want to include something like nova has
+// oc wait --for condition=Ready --timeout=300s Nova/nova
```

Comment:
Perhaps if we want to make sure everything has come up before we go for the RBAC steps below. But for majority of services we don't include such command, we've just been relying on the "post checks" to verify that everything has come up fine (and in tests we have a retry loop on the post checks with some timeout).

Reply:
That was kind of my impression and quandary as well. The API/Conductor shouldn't take long to really start in the grand scheme of the universe. Likely something we just need to get some feedback upon to really make a solid decision upon, and should thus drive towards consistency.